lib/recipes/hs-college-success/bake

```diff
@@ -8,7 +8,6 @@ hs = Kitchen::BookRecipe.new(book_short_name: :hs_college_success) do |doc|
   include Kitchen::Directions
 
   book = doc.book
-  metadata = book.metadata
```

Comment:
Did you mean to remove the metadata?

Reply:
Hey, yes, that line already exists in the main college success recipe which is passed into the hs-college success recipe.